x/fundraising/types/auction.go

```diff
@@ -466,3 +468,12 @@ func (ba BaseAuction) GetMaxBidAmount(bidder string) sdk.Int {
 	}
 	return maxBidAmt
 }
+
+func (ba BaseAuction) SetMaxBidAmount(bidder string, maxBidAmt sdk.Int) error {
```

Comment:
1. Does this also lead to add this method in (type AuctionI interface)?
2. Is "GetMaxBidAmount" also necessary?

Reply:
Yes, let me add those in the docs. Thanks!